testutil/singlevm/setup.sh

```diff
@@ -63,6 +65,11 @@ sudo killall ciao-scheduler
 sudo killall ciao-controller
 sudo killall ciao-launcher
 sudo killall qemu-system-x86_64
+sudo killall dnsmasq
+echo "Original /etc/hosts is temporarily move to $hosts_file_backup"
```

Comment:
@obedmr what if the script hands due to some reason and the user hits Ctrl-C? Then we have potentially messed up the users's system.. may be good to backup the original hosts file at the test directory also